src/cc65/stdfunc.c

```diff
@@ -185,6 +185,15 @@ static void ParseArg (ArgDesc* Arg, Type* Type)
 
 
 
+static void MaybeCodeLine (const char* Code, long IVal)
```

Comment:
Of course, "static" must be removed for MaybeCodeLine() to be usable from the other functions above, too.

For me, the name MaybeCodeLine() is not good, because it does not describe what it actually does. Either use it generically ( void MaybeCodeLine(int GenerateLine, const char* Code, ...) and use it like "if (GenerateLine) { /* printf()-style handling of the arguments)", or rename it to something like "ConditionallyCompareToValueIfNotZero()".